### ISSUE TITLE
【NotFoundError】Key not found in checkpoint

### ISSUE THREAD

**Issue by tamio0800** [JaywongWang/I3D-Feature-Extractor#2]
Hello JaywongWang,

Nice job, it's quite amazing and helpful, yet I have ran into a problem of 'mismatch between checkpoint files and model'.

I downloaded checkpoint files from "https://github.com/deepmind/kinetics-i3d/tree/master/data/checkpoints/rgb_imagenet" as you mentioned, but when I executed "saver.restore(sess, _CHECKPOINT_PATHS['rgb_imagenet'])" from 「feature_extractor_frm.py」 it says :「Key Conv3d_1a_7x7/batch_norm/beta not found in checkpoint
	 [[node save/RestoreV2 (defined at <ipython-input-9-71e34de0891b>:10) ]]」.

Could you please lend me a hand? Thank you very much!

Best,
Tamio

**Comment by JaywongWang** [comment]
I guess it might be an environment issue. I used python 2.7 and tensorflow 1.14.0.

**Comment by Serendipity-LC** [comment]
Hi！How did you solve this problem？

**Comment by tamio0800** [comment]
Dear JaywongWang  and Gray-3 ,

Sorry for the late reply, I had resolved this problem by following JaywongWang's instruction -- rebuilding my environment with Python 2.7 and Tensorflow-gpu 1.14.0.

Thanks for your help again!!

**Comment by DeepRunner** [comment]
> Dear JaywongWang and Gray-3 ,
> 
> Sorry for the late reply, I had resolved this problem by following JaywongWang's instruction -- rebuilding my environment with Python 2.7 and Tensorflow-gpu 1.14.0.
> 
> Thanks for your help again!!

Hi, I change the environment with python2.7 and tf-gpu 1.14.0. However I still met the same problem. What is your sonnet and cuda version?

**Comment by DeepRunner** [comment]
By the way, have you achieved accuracy with the extracted feature on the Charades of Temporal Grounding Task?
> Dear JaywongWang and Gray-3 ,
> 
> Sorry for the late reply, I had resolved this problem by following JaywongWang's instruction -- rebuilding my environment with Python 2.7 and Tensorflow-gpu 1.14.0.
> 
> Thanks for your help again!!

**Comment by twmoveon** [comment]
Any updates on this? I'm running under the same environment but restoring issue still exists.
